index.html

```diff
@@ -0,0 +1,304 @@
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <section class="works">
+        <h3 id="wors_descp">Professional Art Printing Data</h3>
+        <p id="work_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <section class="works">
+        <h3 id="wors_descp">Professional Art Printing Data</h3>
+        <p id="work_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <section class="works">
+        <h3 id="wors_descp">Professional Art Printing Data</h3>
+        <p id="work_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <section class="works">
+        <h3 id="wors_descp">Professional Art Printing Data</h3>
+        <p id="work_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+        <article class="lang_container_2">
+            <ul style="list-style-type:none;" class="list">
+                <li id="lang1">HTML</li>
+                <li id="lang2">Bootstrap</li>
+                <li id="lang3">Ruby</li>
+            </ul>
+        </article>
+        <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+            Project</button>
+    </section>
+    <!-- mobile projects section end  -->
```

Comment:
- [ ] Just like @mckent05 has pointed out, what you have done here could not be considered right or best practice. In responsive design, we need to work with the same content but use media queries to lay the content responsively for smaller and larger screens. Please follow the suggestion that @mckent05 left above.